integration/go-grpc-client/interop_test.go

```diff
@@ -0,0 +1,116 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package interop
+
+import (
+	"bytes"
+	"log"
+	"os"
+	"reflect"
+	"testing"
+
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+
+	"go.opencensus.io/plugin/ocgrpc"
+	"go.opencensus.io/tag"
+	"go.opencensus.io/trace"
+
+	pb "github.com/census-instrumentation/opencensus-experiments/integration/proto"
+)
+
+var setups = []struct {
+	name         string
+	envAddrKey   string
+	fallbackAddr string
+}{
+	{name: "GoClient-GoServer", envAddrKey: "GO_SERVER_ADDR", fallbackAddr: ":9800"},
```

Comment:
kinda weird that this is being configured in Go: i would expect the server/client combination to be specified at the top level in a shell script / makefile so other languages wouldn't need to be concerned with editing Go files